src/PSR7/Validator.php

```diff
@@ -126,7 +163,7 @@ protected function findPathSpec(PathAddress $addr) : PathItem
      * This should consider path parameters as well
      * "/users/12" should match both ["/users/{id}", "/users/{group}"]
      *
-     * @param ServerRequest $request
+     * @param ServerRequestInterface $request
```

Comment:
Updated according to signature

Reply:
The package obeys to this coding style: https://github.com/doctrine/coding-standard
There is a git hook, with formats the code accordingly to the standard on each commit.
It will strip such comments off, because they add questionable value as the type already expressed in the signature.

Reply:
I've added a chapter to Readme file: `Contribution Guide`, please take a look.

Reply:
Ok, so should I juts remove this line ?

Reply:
I think to ensure CS consistency even withouth making contributors installing hooks a travis script might be added making PR check fail if CSC are not kept

Reply:
I've applied `phpcbf` to this file